pysat/utils/io.py

```diff
@@ -800,7 +810,7 @@ def load_netcdf(fnames, strict_meta=False, file_format='NETCDF4',
                                         epoch_name=epoch_name,
                                         epoch_unit=epoch_unit,
                                         epoch_origin=epoch_origin,
-                                        labels=labels,
+                                        meta_kwargs=meta_kwargs, labels=labels,
```

Comment:
may want to be careful adding keywords in the middle of a list. Users don't have to specify keywords in calls, order can be sufficient. Adding things in the middle can break existing code that relies upon order.

Reply:
Leaning toward keeping the order grouping for ease of reading, especially with this many kwargs.  @aburrell?

Reply:
I put this here so that it is close to other relevant kwargs.  If you all want to change it I don't care.

Reply:
Looks like two votes in favor of keeping this order.  We definitely changed the order already when the extra epoch kwargs were added.